homeassistant/components/eufy/manifest.json

```diff
@@ -4,6 +4,6 @@
   "codeowners": [],
   "documentation": "https://www.home-assistant.io/integrations/eufy",
   "iot_class": "local_polling",
-  "loggers": ["lakeside"],
-  "requirements": ["lakeside==0.12"]
+  "loggers": ["lakeside2"],
+  "requirements": ["lakeside2==0.13"]
```

Comment:
Did you ask the maintainer of lakeside if they would want to transfer the repo and pypi project to you?

If that's not an option, please disconnect your fork from upstream to make it clear to the community that your repo is now upstream for the project.

Reply:
I did ask him and he said he'd be happy to transfer ownership but doesn't have the ability to easily transfer the github repo.  He did say he could give me the pypi ownership.  I'm not sure if it makes sense to go that route though.  What do you think?

I have disconnected my fork from the upstream.

Reply:
The benefit of transferring the PyPI package is that users of the library can just upgrade to a later version instead of having to find the new library. So I think that's good. If you do that, make sure you update the link to the repo, via the package settings, on your next release.